app/hooks/useCreateTodo.jsx

```diff
@@ -0,0 +1,37 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import httpService from '../core/http-config';
+
+export const useCreateTodo = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: async (newTask) => {
+      const response = await httpService.post('/todos', newTask);
+      return response.data;
+    },
```

Comment:
Create a separated function for create task like delete and update @sadikshya2001